addons/transform/org.openhab.transform.feed/README.MD

```diff
@@ -0,0 +1,59 @@
+# Feed Transformation Service
+
+Feed transformation service transforms a feed between some of the supported feed formats: RSS 0.9, RSS 0.91 Netscape, RSS 0.91 Userland, RSS 0.92, RSS 0.93, RSS 0.94, RSS 1.0, RSS 2.0, Atom 0.3, Atom 1.0.
+
+## Example
```

Comment:
Personally I have no clue how I can and or should use it and what the use case is, I looked at https://github.com/openhab/openhab1-addons/wiki/Transformations but that did not make it clear to me either. But let it be between you and @kaikreuzer